tests/DeadlineDateServiceTest.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+require_once 'src/Services/DeadlineDateService.php';
+
+class DeadlineDateServiceTest extends \PHPUnit\Framework\TestCase
```

Comment:
A very minor thing with these tests - there are specific phpunit assertions for testing booleans, worth using them to make the code a bit more compact

So instead of doing

```
$expected = false;
$this->assertEquals($expected, $actual);
```

You can do:
```
$this->assertFalse($actual);
```

There is also an assertTrue you can use in your overdueTrue test